Source/ocdm/adapter/gstreamer/open_cdm_adapter.cpp

```diff
@@ -39,6 +39,19 @@ inline bool mappedBuffer(GstBuffer *buffer, bool writable, uint8_t **data, uint3
     return true;
 }
 
+uint32_t opencdm_construct_session_private(struct OpenCDMSession* session, void* &pvtData)
```

Comment:
I guess this is not needed as the static will look for these methods and if the methods are not there just set the pointer to null ???? Right ?

Reply:
Yes, I agree. Removed